config/settings.yml

```diff
@@ -1121,6 +1121,11 @@
       :event_handler:
         :cpu_usage_threshold: 0.percent
         :nice_delta: 7
+        :queue_type: artemis
+        :queue_hostname: localhost
+        :queue_port: 61616
+        :queue_username: admin
+        :queue_password: smartvm
```

Comment:
Again, this should be passed via env vars.  @carbonin can give you the specifics, once we work out how the container will be launched in kubernetes.